### PR TITLE
Self-hosted v23.1.15 release notes

### PR DESCRIPTION
**Pull request by Amruta-Ranade**

### CODE REVIEW

File: src/current/_data/releases.yml [hunk 1]
```diff
@@ -5470,34 +5470,27 @@
...
-    [contact support](https://support.cockroachlabs.com/hc/en-us/requests/new).
   mac:
     mac_arm: true
     mac_arm_experimental: true
     mac_arm_limited_access: false
   windows: true
   linux:
     linux_arm: true
-    linux_arm_experimental: false
+    linux_arm_experimental: true
```
Comment (Amruta-Ranade) [inline src/current/_data/releases.yml:5486]: @mdlinville Should this be true?

File: src/current/_data/releases.yml [hunk 2]
```diff
@@ -5470,34 +5470,27 @@
...
     linux_arm_limited_access: false
     linux_intel_fips: true
     linux_arm_fips: false
   docker:
     docker_image: cockroachdb/cockroach
     docker_arm: true
-    docker_arm_experimental: false
+    docker_arm_experimental: true
```
Comment (Amruta-Ranade) [inline src/current/_data/releases.yml:5493]: @mdlinville Should this be true?

File: src/current/_data/releases.yml (hunk 1 above)
Comment (mdlinville) [inline src/current/_data/releases.yml:5486]:
```suggestion
    linux_arm_experimental: false
```

File: src/current/_data/releases.yml (hunk 2 above)
Comment (mdlinville) [inline src/current/_data/releases.yml:5493]:
```suggestion
    docker_arm_experimental: false
```

**Review by mdlinville — APPROVED** [review]

**Pull request merged**